packages/create-ponder/src/helpers/graphqlTranslator.ts

```diff
@@ -0,0 +1,265 @@
+import { gql } from "graphql-tag";
```

Comment:
Any reason not to use `parse` from `graphql`? Looks like `graphql-tag` uses it under the hood anyway.

Reply:
No real reason, it was just the first thing that I found that works